src/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/StreamingContext.cs

```diff
@@ -18,60 +18,54 @@ namespace System.Runtime.Serialization
 {
     public struct StreamingContext
     {
-        internal Object m_additionalContext;
-        internal StreamingContextStates m_state;
+        internal Object _additionalContext;
+        internal StreamingContextStates _state;
 
-        internal StreamingContext(StreamingContextStates state)
+        public StreamingContext(StreamingContextStates state)
             : this(state, null)
         {
         }
 
-        internal StreamingContext(StreamingContextStates state, Object additional)
+        public StreamingContext(StreamingContextStates state, Object additional)
         {
-            m_state = state;
-            m_additionalContext = additional;
-        }
-
-        internal Object Context
-        {
-            get { return m_additionalContext; }
+            _state = state;
+            _additionalContext = additional;
         }
 
+        public Object Context => _additionalContext;
+        
+        public StreamingContextStates State => _state;
+        
         public override bool Equals(Object obj)
         {
             if (!(obj is StreamingContext))
             {
                 return false;
             }
             StreamingContext ctx = (StreamingContext)obj;
-            return ctx.m_additionalContext == m_additionalContext && ctx.m_state == m_state;
+            return ctx._additionalContext == _additionalContext && ctx._state == _state;
         }
 
         public override int GetHashCode()
         {
-            return (int)m_state;
-        }
-
-        internal StreamingContextStates State
-        {
-            get { return m_state; }
+            return (int)_state;
         }
     }
 
     // **********************************************************
     // Keep these in sync with the version in vm\runtimehandles.h
     // **********************************************************
```

Comment:
I would prefer to keep the comment till we decide what to do with the other values, e.g. remoting